src/cargo/sources/registry/remote.rs

```diff
@@ -29,7 +29,6 @@ pub struct RemoteRegistry<'cfg> {
     config: &'cfg Config,
     tree: RefCell<Option<git2::Tree<'static>>>,
     repo: LazyCell<git2::Repository>,
-    head: Cell<Option<git2::Oid>>,
     current_sha: Cell<Option<InternedString>>,
```

Comment:
Couldn't we drop `current_sha` as well?

Reply:
It is never used/set, I will remove it in the next PR.

Reply:
Needed to keep it to bring back the support for pre #11044 cache files.